### ISSUE TITLE
App crash on startup

### ISSUE THREAD

**Issue by johansatge** [namagicapp/support#8]
A few users report that the application crashes on startup with the following error:

```
Crashed Thread:        0  CrBrowserMain  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BREAKPOINT (SIGTRAP)
Exception Codes:       0x0000000000000002, 0x0000000000000000
```
Complete log file is available [here](https://paste.ee/p/SDhk4).

**Comment by johansatge** [comment]
This has been fixed in `1.1.0`, issued today.

**Comment by johansatge** [comment]
This issue was moved to johansatge/namagic#6
